srtcore/list.cpp

```diff
@@ -93,6 +93,20 @@ CSndLossList::~CSndLossList()
     releaseMutex(m_ListLock);
 }
 
+void CSndLossList::traceState() const
+{
+    int pos = m_iHead;
+    while (pos != -1)
+    {
+        ::cout << pos << ":[" << m_caSeq[pos].seqstart;
```

Comment:
It would be better to use `LOG*` facility to display it. Even if this is only left for future debugging.

Of course, you'd need `std::ostringstream` for intermediate collection, as a single log instruction cannot print a fragment of a line.

Reply:
Can LOGF not end the line with `\n` like `Verb()` allows to do?

Reply:
Unfortunately, no. One log instruction always ends with an end of line. Partial-line logging would have to be added support.

Reply:
Then would be simpler to leave this type of output for the sake of code simplicity.

Reply:
`Verb()` has a special `VerbNoEOL` tag to leave without EOL. But `Verb()` doesn't have a log header, so there's a little problem with this. For `LOG*` we'd need something like a temporary log cumulation variable, which will be then reused when doing a final log with EOL. But I don't think we could this way do any better than by using `std::ostringstream`.